src/components/blog/post/body.js

```diff
@@ -0,0 +1,20 @@
+import React from "react"
+import PropTypes from "prop-types"
+
+import BodyWrapper from "./body_wrapper"
+
+import styles from "./body.module.scss"
+
+const PostBody = ({ html }) => (
+  <BodyWrapper>
+    {/* eslint-disable react/no-danger */}
```

Comment:
`// eslint-disable-next-line react/no-danger` :upside_down_face:

Reply:
Nice one, changed in 93dfdaf.